arm-network/2016-06-01/network.json

```diff
@@ -6515,14 +6601,143 @@
       ],
       "description": "NetworkSecurityGroup resource"
     },
+	"EffectiveNetworkSecurityGroup": {
+      "properties": {
+        "networkSecurityGroup": {
+	      "$ref": "#/definitions/SubResource",
+          "description": "Gets the id of network security group that is applied"
+		},
+		"association": {
+		  "$ref": "#/definitions/EffectiveNetworkSecurityGroupAssociation"		  
+		},
+		"effectiveSecurityRules": {
+		  "type": "array",
+          "items": {
+            "$ref": "#/definitions/EffectiveNetworkSecurityRules"
+          },
+          "description": "Gets collection of effective security rules"
+		}
+	  },
+      "description": "Effective NetworkSecurityGroup"	  
+    },
+	"EffectiveNetworkSecurityGroupAssociation": {
+      "properties": {
+        "subnet": {
+	      "$ref": "#/definitions/SubResource",
+          "description": "Gets the id of subnet if assigned"
```

Comment:
sibiling properties of the $ref will be ignored. The Description, should be in the reference definition